code/game/objects/items/manuals.dm

```diff
@@ -433,6 +433,57 @@
 	title = "Toxins or: How I Learned to Stop Worrying and Love the Maxcap"
 	page_link = "Guide_to_toxins"
 
+//NSV13 - Change
+//Standard operating prosedure books. Inherited of wiki books.
+//To do: Get book specific sprites.
+//       Get book for munitions, once page is made / up to date.
+//
+
+/obj/item/book/manual/wiki/sop
+	icon_state = "bookDetective" //remove if there is department specific sprites, add this to subtypes
```

Comment:
Can you see if you can get sprites for these as they are so specific its a extremely good idea for it long term so everyone knows what each book is.

Reply:
All sprites are in now. A munitions book can be created once we got muni sop!